Generative-AI/Youtube_Rag_Chatbot/main.py

```
@@ -0,0 +1,125 @@
+from youtube_transcript_api import YouTubeTranscriptApi, TranscriptsDisabled, VideoUnavailable
+from langchain_community.embeddings import HuggingFaceEmbeddings
+from langchain_community.vectorstores import FAISS
+from langchain_core.documents import Document
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain_core.prompts import PromptTemplate
+from langchain_google_genai import ChatGoogleGenerativeAI
+from langchain_core.runnables import RunnableParallel, RunnablePassthrough, RunnableLambda
+from langchain_core.output_parsers import StrOutputParser
+from typing import Annotated
+from dotenv import load_dotenv
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)